aw_transform/classify.py

```diff
@@ -14,7 +14,7 @@ class Rule:
 
     def __init__(self, rules: Dict[str, str]):
         if "regex" in rules:
-            self.regex = re.compile(rules["regex"])
+            self.regex = re.compile(rules["regex"]) if rules["regex"] else None
```

Comment:
Spaghetti code, first you check if "regex" is in rules and then again you check if regex is in rules.

Would make more sense to write

        if "regex" in rules:
            self.regex = re.compile(rules["regex"])
        else:
            self.regex = None

Oh, now that I think of it it's even incorrect as well as "regex" is impossible to not be set and it is possible to set rules["regex"] = False which would be silly.

Reply:
It's not actually, the check is for when the regex string is an empty string, which would erroneously match everything. And regex might not be set at all, in the future another rule might be used instead.